modules/mvt/src/lib/binary-vector-tile/vector-tile-feature.ts

```diff
@@ -171,7 +190,7 @@ export default class VectorTileFeature {
     const x0 = this.extent * x;
     const y0 = this.extent * y;
 
-    function project(data) {
+    function project(data: any[]) {
```

Comment:
Move this function outside of the class. It would be great to create a `helper` file for outer functions.